tests/attestation_config.rs

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use attestation::snp::PolicyFlags;
+use attestation::{Digest, Measurements};
+use steward::Config;
+
+use std::collections::HashSet;
+
+use der::Decode;
+use sgx::parameters::MiscSelect;
+use x509::request::CertReq;
+
+mod common {
+    pub mod config;
+    pub mod validation;
+}
+use common::config::{assert_sgx_config, DEFAULT_CONFIG, ICELAKE_CSR};
+
+#[test]
+fn test_config_empty() {
+    let config: Config = toml::from_str("").expect("Couldn't deserialize");
+
+    assert!(config.sgx.is_none());
+    assert!(config.snp.is_none());
+}
+
+#[test]
+fn test_config() {
```

Comment:
This is an isolated unit tests, which should either be:
- extended to run the full Steward and process requests
- dropped
- moved close to `Config` type implementation

Reply:
File deleted